docker-compose.dev.yaml

```diff
@@ -4,7 +4,7 @@ services:
   point_node:
     image: ${POINT_DEV_IMAGE_NAME:-pointnetwork/pointnetwork_node:$POINT_NODE_VERSION}
     extends:
-      file: docker-compose.yaml
+      file: docker-compose.base.yaml
```

Comment:
Ok. In this case I needed also to make zappdev compose file extend base and not dev version of compose since the ports will be expose in dev and cannot be overriden in zappdev. This caused the volumes of dev being replicated in the zappdev compose.

Reply:
@tankakatan this is not a feature implemented in docker compose, is only a solution proposal from a person in the thread. I just tried and does not even run the compose command with syntax error. I read this thread and other ones about override/exclude ports in docker compose and currently there is way to do that. The default behavior for ports when extending is to concatenate and not override, as stated in the official docs of docker compose [here](https://docs.docker.com/compose/extends/#adding-and-overriding-configuration).

Reply:
Thanks @diogosmendonca. Then we have to either introduce another "base" compose file with local dirs binding or copy and paste this binding to `arlocal` compose file. Both options are not very great. Maybe I will find another solution in the future. But the fact is we need to run the `dev` compose separately. That is why we need the port binding there.